ecs-image-build/Dockerfile

```diff
@@ -0,0 +1,8 @@
+ARG IMAGE_VERSION="latest"
+FROM 416670754337.dkr.ecr.eu-west-2.amazonaws.com/ci-node-runtime-18:${IMAGE_VERSION} 
```

Comment:
Possibly the ecs build can inject it's own Image_version in case it needs to be other than latest. We copied this from confirmation-statement-web so I think it's safe.